js/views/userPageVw.js

```diff
@@ -381,6 +381,8 @@ module.exports = Backbone.View.extend({
       processData: false,
       data: formData,
       success: function(data) {
+        var errorModal = $('.js-messageModal');
+        errorModal.removeClass('hide');
```

Comment:
Good point. I saw it in the if and the else though, so I may have been mistaken thinking it should just be done arbitrarily.